Python 3.12.1 (tags/v3.12.1:2305ca5, Dec  7 2023, 22:03:25) [MSC v.1937 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 16, in <module>
    main()
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 15, in main
    calcAreaofCircle(4)
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 3, in calcAreaofCircle
    pi = math.pi
NameError: name 'math' is not defined. Did you forget to import 'math'?

============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 17, in <module>
    main()
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 16, in main
    calcAreaofCircle(4)
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 5, in calcAreaofCircle
    result = pi*r*r
NameError: name 'r' is not defined

============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
4
4
129\
125
SyntaxError: invalid syntax
>>> 8
8
>>> 12
12
>>> 10
10
>>> 
============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
<function calcAreaofCircle at 0x000001835BC8F1A0>
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 18, in <module>
    main()
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 17, in main
    print(PerimeterOfCircle)
NameError: name 'PerimeterOfCircle' is not defined. Did you mean: 'calcPerimeterofCircle'?
>>> 
============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
<function calcAreaofCircle at 0x000001DD6064F1A0>
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 18, in <module>
    main()
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 17, in main
    print(calcPerimeterOfCircle)
NameError: name 'calcPerimeterOfCircle' is not defined. Did you mean: 'calcPerimeterofCircle'?
>>> 
============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
<function calcAreaofCircle at 0x0000020EFF77F1A0>
<function calcPerimeterofCircle at 0x0000020EFF77F100>
>>> 
============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 23, in <module>
    main()
  File "C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py", line 16, in main
    calcAreaOfCircle(12)
NameError: name 'calcAreaOfCircle' is not defined. Did you mean: 'calcAreaofCircle'?
>>> 
============================================== RESTART: C:/Users/User/Desktop/Dalitso Excercise/myGeometryDalitso.py ==============================================
<function calcAreaofCircle at 0x00000259E174F1A0>
<function calcPerimeterofCircle at 0x00000259E174F100>
